src/DwitTech.AccountService.Data/Entities/ValidationCode.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+using DwitTech.AccountService.Core;
+using DwitTech.AccountService.Core.Enums;
+using Microsoft.EntityFrameworkCore;
+
+namespace DwitTech.AccountService.Data.Entities
+{
+    public class ValidationCode : BaseEntity
+    {
+        public string UserId { set; get; }
```

Comment:
UserId cannot be a string because it's of int type in the User entity. This will likely cause conflicts. Please change it to int.

Reply:
Why do you think it shouldn't be a string?